src/main/java/org/jenkinsci/plugins/workflow/libs/SCMRetriever.java

```diff
@@ -72,7 +74,7 @@ public SCM getScm() {
             return "Legacy SCM";
         }
 
-        public static List<SCMDescriptor<?>> getSCMDescriptors() {
+        public List<SCMDescriptor<?>> getSCMDescriptors() {
```

Comment:
🐛 Binary compatibility change. Please keep [and deprecate] the original method and implement a new one if required